src/hooks/useVibeMediaQuery/__stories__/useVibeMediaQuery.stories.js

```diff
@@ -0,0 +1,16 @@
+import useVibeMediaQuery from "..";
+import "../../__stories__/general-hooks-stories.scss";
+import styles from "./useVibeMediaQuery.modules.scss";
+
+export default {
+  title: "Hooks/useVibeMediaQuery"
+};
+
+export const Overview = {
+  render: () => {
+    const currentSize = useVibeMediaQuery();
+    return <div className={`hooks-reference-element ${styles.container}`}>{currentSize}</div>;
+  },
+
+  name: "Overview"
```

Comment:
Isn't it redundant? It takes the function's name

Reply:
It is mostly
It's automatic converter job
Though I guess there are cases when it's needed